src/App.tsx

```diff
@@ -53,7 +53,7 @@ const App = () => {
   const dispatch = useDispatch();
   const { t, i18n } = useTranslation('feedback');
   const user = useSelector((store: RootStore) => store.user);
-  const [isLoadingUser, setIsLoadingUser] = useState(true);
+  const [isLoadingUser, setIsLoadingUser] = useState({ attributes: true, roles: true, authority: true });
```

Comment:
hva er attributes?

Reply:
skjønner ikke hvor disse brukes? hvorfor settes de?

Reply:
er det denne? `{Object.values(isLoadingUser).some((isLoading) => isLoading)` hvis en av de er true, så rendres det?

Reply:
isåfall, så ønsker jeg meg mer beskrivende navn 😅

Reply:
tenker du å bruke noen av disse feltene til noe i fremtiden? eller hvorfor endret du fra en enkel bool til objekt?

Reply:
ja, tidligere var det bare en `isLoadingUser`, men den var et resultat av 3 oppslag: `getCurrentUserAttributes` (Feide-bruker), `getInstitutionUser` (roller), og `getAuthority`. Disse ga ikke nødvendigvis svar i samme rekkefølge hver gang, så det hendte at `isLoading` ble satt til `false` selv om det fortsatt var ting in progress

Er dette bedre?
`const [isLoading, setIsLoading] = useState({ userAttributes: true, userRoles: true, userAuthority: true });`

Reply:
I see 👍  ja - jeg synes det er bedre sånn der!